README.md

```diff
@@ -447,7 +447,6 @@ Below are the major development axis identified for Dyna&omega;o for the next fe
 
 ### Axis 3 - Dependencies, architecture and performances
 * ~~Switch to OpenModelica V1.13 version and DAE mode use~~
-* Switch to Sundials V5.0.0 and Suitesparse V5.3.0
```

Comment:
We generally strike out the text and add the version in which it is added instead of removing it

Reply:
OK